README.rst

```diff
@@ -26,6 +26,14 @@ then run the following command to install the other packages that the draft requ
 
    sudo tlmgr install latexmk isodate substr relsize ulem fixme rsfs extract layouts enumitem
 
+-------------------------
+Getting Started on Ubuntu
```

Comment:
Happy to change it however you'd prefer. Would you like me to go with Debian-based or Debian?